swap_meet/decor.py

```diff
@@ -0,0 +1,15 @@
+from .item import Item
+class Decor(Item):
+    def __init__(self, condition =0):
```

Comment:
*annoying nitpick, disregard me*

```suggestion
    def __init__(self, condition = 0):
```